src/js/components/homepage/AwardSearch/AwardSearch.jsx

```diff
@@ -4,14 +4,25 @@
  */
 
 import React from 'react';
-import { FlexGridRow } from 'data-transparency-ui';
+import { FlexGridRow, FlexGridCol } from 'data-transparency-ui';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 
 const AwardSearch = () => (
     <section className="award-search__section">
         <FlexGridRow className="grid-content" >
-            <h1>
-            AWARD SEARCH COMPONENT
-            </h1>
+            <FlexGridCol mobile={12}>
```

Comment:
I'm still seeing the green border for this row.  The width on mobile is correct for this section but not on desktop.  Are you planning to take care of both of these in the carousel ticket?

Reply:
Ah - i left the green border for the time being since it helped me find the section, I indeed adjusted the width on desktop when i added the carousel

i should merge qat in though bc i think there was a site max width change that brian added